client/pkg/git/git_test.go

```diff
@@ -7,9 +7,15 @@ import (
 func TestRemoteURL(t *testing.T) {
 	t.Parallel()
 
+	actual := RemoteURL("example.com", "app")
 	expected := "ssh://git@example.com:2222/app.git"
 
-	actual := RemoteURL("example.com", "app")
+	if actual != expected {
+		t.Errorf("Expected %s, Got %s", expected, actual)
+	}
+
+	actual = RemoteURL("deis.10.245.1.3.xip.io:31350", "velcro-underdog")
```

Comment:
we should also keep the original test for "example.com" so we can ensure that remote URLs with no port number don't error out :)